automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainOwnersSpec.scala

```diff
@@ -26,7 +26,7 @@ import scala.util.Try
  *
  * TODO: Fix Rawls group creation/deletion and run these tests in parallel
  */
-class AuthDomainOwnersSpec extends FreeSpec /*with ParallelTestExecution*/ with Matchers
+class AuthDomainOwnersSpec extends FreeSpec with ParallelTestExecution with Matchers
```

Comment:
Has rawls been fixed as noted in the TODO above?

Reply:
per @MatthewBemis, answer is yes.

Reply:
Then the comment should be removed.